frontend/src/components/DataCocktail.jsx

```diff
@@ -0,0 +1,62 @@
+const dataCocktail = [
```

Comment:
more cocktailsData, isn't it ? or just cocktails ?

this isn't a components, so it should not be in the components folder (and it's not in jsx) : a `data` folder in src ? named cocktails.js